modules/search/client/components/UsersList.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Avatar = ({ avatarSize, avatarSource, userData }) => (
+const Avatar = ({ avatarSize, avatarSource, displayName, profileUrl }) => (
```

Comment:
Let's start this as an external component so that it becomes reusable (we'll literally need it everywhere from now on).

Our previous client-side Avatar directive is terrible; we should clearly move all that logic to the backend:

https://github.com/Trustroots/trustroots/blob/bbacc11d05923134a8fc140472473b43ded17716/modules/users/client/directives/tr-avatar.client.directive.js

Reply:
See #997

Reply:
https://github.com/Trustroots/trustroots/pull/997 is now merged and this can be changed to use new Avatar component instead. 💪